app/views/newflow/educator_signup/educator_change_signup_email_form.html.erb

```diff
@@ -1,8 +1,17 @@
+<%
+    current_step = I18n.t(
+        :"login_signup_form.step_counter",
+        current_step: 2,
+        total_steps: @total_steps
+    )
```

Comment:
the current step is another one of those pieces of state data that would be good to expose to the views thru a data object as a PORO.   fine i guess for here...but just sayin

Reply:
Right; every single variable in the view(s) between `<%` and `%>` CAN be eventually put into a ViewModel — as part of the "Model–view–viewmodel" concept.

At this time, the whole Accounts app is an MVC though. Future improvement for sure 👍 .